docs/sources/alerting/set-up/provision-alerting-resources/terraform-provisioning/index.md

```diff
@@ -0,0 +1,316 @@
+        mute_timings = [grafana_mute_timing.my_mute_timing.name]
+
+        policy {
+            matcher {
+                label = "sublabel"
+                match = "="
+                value = "subvalue"
+            }
+            contact_point = grafana_contact_point.a_third_contact_point.name
+            group_by = ["..."]
+        }
+    }
+
+}
+
+1. In the mute_timings field, link a mute timing to your notification policy.
+
+1. Run the command ‘terraform apply’.
+
+1. Go to the Grafana UI and check the details of your notification policy.
+
+**Note:**
+
+You cannot edit resources provisioned from Terraform from the UI. This ensures that your alerting stack always stays in sync with your code.
+
+1. Click **Test** to verify that the notification point is working correctly.
+
+## Provision mute timings
+
+Mute timings provide the ability to mute alert notifications for defined time periods.
+
+To provision mute timings, complete the following steps.
+
+1. Copy this code block into a .tf file on your local machine.
+
+In this example, alert notifications are muted on weekends.
+
+resource "grafana_mute_timing" "my_mute_timing" {
+name = "My Mute Timing"
+
+    intervals {
+        times {
+          start = "04:56"
+          end = "14:17"
+        }
+        weekdays = ["saturday", "sunday", "tuesday:thursday"]
+        months = ["january:march", "12"]
+        years = ["2025:2027"]
+    }
+
+}
+
+1. Run the command ‘terraform apply’.
+1. Go to the Grafana UI and check the details of your mute timing.
+
+**Note:**
+
+You cannot edit resources provisioned from Terraform from the UI. This ensures that your alerting stack always stays in sync with your code.
+
+1. Click **Test** to verify that the mute timing is working correctly.
+
+## Provision alert rules
+
+[Alert rules](https://grafana.com/docs/grafana/latest/alerting/alerting-rules/) enable you to alert against any Grafana data source. This can be a data source that you already have configured, or you can [define your data sources in Terraform](https://registry.terraform.io/providers/grafana/grafana/latest/docs/resources/data_source) alongside your alert rules.
+
+To provision alert rules, complete the following steps.
+
+1. Create a data source to query and a folder to store your rules in.
+
+In this example, the [TestData](https://grafana.com/docs/grafana/latest/datasources/testdata/) data source is used.
+
+```terraform
+resource "grafana_data_source" "testdata_datasource" {
+    name = "TestData"
+    type = "testdata"
+}
+
+resource "grafana_folder" "rule_folder" {
+    title = "My Rule Folder"
+}
+```
+
+1. Define an alert rule.
+
+For more information on alert rules, refer to [how to create Grafana-managed alerts](https://grafana.com/blog/2022/08/01/grafana-alerting-video-how-to-create-alerts-in-grafana-9/).
+
+1. Group your alert rules.
+
+In this example, the `grafana_rule_group` resource group is used.
```

Comment:
I'm not sure if this sentence adds much. You cannot create an un-grouped rule - Terraform _forces_ you to define rules in the form of rule groups, so the grafana_rule_group resource will be used every time.

Maybe we could word it something like `Create a rule group containing one or more rules.`?